.github/workflows/worker.yml

```diff
@@ -0,0 +1,36 @@
+name: worker
+on:
+  push:
+    branches:
+      - "master"
+      - "worker/**"
+
+jobs:
+  binary:
+    runs-on: windows-latest
+
+    steps:
+      - name: 🐙 Checkout GitHub repo (+ download lfs dependencies)
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: 🐍 Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+
+      - name: 🐍 Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install -r requirements-dev.txt
+
+      - name: 📦 Packaging AntaresWebWorker
+        run: bash ./package_worker.sh
+        working-directory: scripts
+
+      - name: 🚀 Upload binary
+        uses: actions/upload-artifact@v3
```

Comment:
You can use `actions/upload-artifact@v4`. See: https://github.com/actions/upload-artifact